configs/server/forseti_conf_server.yaml.sample

```diff
@@ -102,6 +102,8 @@ scanner:
           enabled: false
         - name: ke_version_scanner
           enabled: true
+        - name: service_account_key_scanner
```

Comment:
This should be service_account_key instead of service_account_key_scanner

Reply:
good catch, fixed!